qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -236,8 +248,15 @@ def run_test(self):
             'AllowRevealedSenders',
             'AllowLinkingAccountAddresses',
         ]:
+            if policy == 'FullPrivacy':
+                inner_revealed_recipients_msg = revealed_recipients_msg
+            else:
+                inner_revealed_recipients_msg = """This transaction cannot be sent as is. The following issues were identified:
+* This transaction would have transparent recipients, which requires AllowRevealedRecipients because it will publicly reveal amounts and addresses of the transaparent funds.
```

Comment:
```suggestion
* This transaction would have transparent recipients, which requires AllowRevealedRecipients because it will publicly reveal amounts and addresses of the transparent funds.
```